packages/layout/src/components/es-empty-state/readme.md

```diff
@@ -0,0 +1,69 @@
+# es-empty-state
+
+
+
+<!-- Auto Generated Below -->
+
+
+## Overview
+
+Display an empty state with an illustration and a message.
+Intended for use as `Page`'s `renderEmptyState`.
+
+## Usage
+
+### Example
+
+```tsx
+
+export default () =>
+<div>
+    <es-empty-state header={'Example empty state'} body={<es-button>{'There are no items'}</es-button>}>
+    <es-illustration-group slot={'illustration'} />
+    <p slot='foot'>{ 'Footer goes here' }</p>
+    </es-empty-state>
+    <hr/>
+    <es-empty-state header={'Example empty state'} body={'No items to show'} layout='horizontal'>
+    <es-illustration-group slot={'illustration'} />
+    <p slot='foot'>{ 'Footer goes here' }</p>
+    </es-empty-state>
+</div>
+        
+```
+
+
+
+## Properties
+
+| Property              | Attribute | Description                    | Type                         | Default      |
+| --------------------- | --------- | ------------------------------ | ---------------------------- | ------------ |
+| `header` _(required)_ | `header`  | The header of the empty state. | `string`                     | `undefined`  |
+| `layout`              | `layout`  | The layout of the empty state. | `"horizontal" \| "vertical"` | `'vertical'` |
+
+
+## Shadow Parts
+
+| Part      | Description |
+| --------- | ----------- |
+| `"body"`  |             |
```

Comment:
The parts need to be documented, you should get a warning about it in the build, but I think NX hides it. You can see it on the CI:
https://github.com/EventStore/Design-System/actions/runs/9559092831/job/26348872556?pr=375#step:8:128

To document them, put:

```
@part name - description
```

in the top level doc comment.

Example:
https://github.com/EventStore/Design-System/blob/6bcc2ceb533a4b9c1196e3601156628a31ad72be/packages/components/src/components/es-tabs/es-tabs.tsx#L23